01-Data-Ecosystem/Chapter_1.3_Rows_and_the_Observational_Unit.ipynb

```diff
@@ -965,7 +965,7 @@
         "colab_type": "text"
       },
       "source": [
-        "3\\. What column seems to be appropriate index for this data set? Do you see any problems with using this column as the index? (_Hint:_ Try looking up \"Kelly, Mr. James\" or \"Green, Mr. George\" in this `DataFrame`.)"
+        "3\\. What column seems to be appropriate index for this data set? Do you see any problems with using this column as the index? (_Hint:_ Try looking up \"Kelly, Mr. James\" in this [passenger list](https://en.wikipedia.org/wiki/Passengers_of_the_Titanic#Passenger_list).)"
```

Comment:
The reader hasn't set the `name` column as the index yet (they are instructed to do so in the next step), and doesn't know how to look up a passenger by name otherwise. I recently found this a bit confusing when I worked through this, and realized later that I had noted the same thing when I worked through this in 2022, so I thought it would be worth mentioning.